docs/INTRODUCTION.md

```diff
@@ -1,63 +1,57 @@
 
-## Concepts
+[**OVE Services**](https://github.com/ove/ove-services) provides core functionality microservices within OVE such as the [Layout Service](https://github.com/ove/ove-services/tree/master/packages/ove-service-layout), which is used to compute absolute positions in pixels from positions expressed relative to a grid or as a percentage of the total space size. [**OVE Asset Services**](https://github.com/ove/ove-asset-services) provides an Asset Manager along with a collection of Asset Processing Services, which are used to upload archives or divide large networks or images into tiles, and serve these to the corresponding applications.
 
-OVE supports multiple ``spaces``. Each ``space`` is associated with one or more monitors, which may be attached to different computers, that together form a single display. This display may be a Large High Resolution Display (the [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution and consists of 64 monitors), but OVE is also suitable for use on much smaller displays.
+OVE also provides a [graphical editor](https://github.com/ove/ove-editor) and [software development kits](https://github.com/ove/ove-sdks) that can be used to design and develop projects.
 
-In each space, OVE runs within a number of ``clients``. Each ``client`` is a browser window, typically full-screen on a single monitor, but a single ``client`` can span multiple windows if they are attached to the same computer. The arrangement of ``clients`` is described in the [``Spaces.json``](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
+## Basic concepts
 
-``Sections`` are rectangular regions within an ``oveCanvas``. Each ``section`` runs a single OVE ``app``. ``Sections`` may span multiple clients and can overlap.
+An OVE `server` installation supports multiple `spaces`. A `space` is a collection of monitors, which may be attached to different computers, that together form a single display. OVE is designed to be used in Large High Resolution Display environments; but, it is also suitable for use on much smaller displays with a single or a few monitors.
 
-An ``app`` displays something on the display. The official **core** ``apps`` each provide a way to display a distinct form of commonly used content.
-They include an ``HTML app`` that can be used to display any visualization that run as web apps (subject to the limitations described in the [list of pitfalls to avoid](./PITFALLS.md)). 
+In each space, OVE runs within a number of `clients`. An OVE `client` is a browser window and typically runs full-screen on a single `screen`. A `screen` can be a single monitor or can span multiple monitors if they were all attached to a single computer. The arrangement of `clients` is described in the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
```

Comment:
```suggestion
In each space, OVE runs within a number of `clients`. An OVE `client` is a browser window and typically runs full-screen on a single `screen`. A `screen` can span a single monitor or can span multiple monitors that are attached to the same computer. The arrangement of `clients` is described in the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
```

Reply:
Susan agrees that "that are attached to the same computer" is preferable. She also pointed out that the `screen` is not actually 'a single monitor', but rather is **on** it.

Reply:
I do accept the typo WRT `span` instead of `be`. 

But, there is no gain in debating over the two alternatives, :) for the rest of this line. They would mean the same thing to the reader. I have chosen your alternative, and ending this debate, along the lines of productive use of time.